app/src/main/java/org/fossasia/susi/ai/data/UtilModel.kt

```diff
@@ -23,7 +24,8 @@ class UtilModel(val context: Context) : IUtilModel {
     override fun saveToken(response: Response<LoginResponse>) {
         val loginResponse = response.body()
         if (loginResponse != null) {
-            PrefManager.putString(Constant.ACCESS_TOKEN, loginResponse.accessToken as String)
+            val token = loginResponse.accessToken
+            if (token is String) PrefManager.putString(Constant.ACCESS_TOKEN, token)
```

Comment:
What type is accessToken actually?

Reply:
OK, it is a nullable string. Just make PrefManager accept nullables. Then there'll be no need of this cast